packs/feat-effects/effect-unstable-check-failure.json

```diff
@@ -0,0 +1,36 @@
+{
+    "_id": "olpkQDGDzmYZCvQH",
+    "img": "icons/magic/fire/explosion-fireball-large-purple-orange.webp",
+    "name": "Effect: Unstable Check Failure",
+    "system": {
+        "description": {
+            "value": "<p>On a failure, the innovation malfunctions in a spectacular (though harmless) fashion, such as a belch of smoke or shower of sparks, and it becomes incapable of being used for further unstable actions.</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "encounter",
+            "value": -1
+        },
+        "level": {
+            "value": 0
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Guns & Gears"
+        },
+        "rules": [],
```

Comment:
Shouldn't this effect do something? Perhaps have it add to the description of unstable actions with a reminder that they can't perform them while their innovation is malfunctioning.

Reply:
Good point.

Let me add that.

Reply:
Check again when you get a moment

Reply:
Looks ok to me!